components/VcItemContent.tsx

```diff
@@ -13,33 +13,43 @@ import testIDProps from '../shared/commonUtil';
 const getDetails = (arg1, arg2, verifiableCredential) => {
   if (arg1 === 'Status') {
     return (
-      <Column>
+      <Column
+        style={{
+          justifyContent: 'space-between',
+          alignItems: 'center',
+          marginTop: 7,
+        }}>
         <Text
           testID="status"
-          weight="bold"
-          size="smaller"
+          weight="regular"
+          style={Theme.Styles.titleTextSize}
```

Comment:
Already there is an object {Themes.TextStyles.smaller}.
Every time better to do a back check with styles.

Reply:
used the existing styles